webrtc-stats.html

```diff
@@ -1083,7 +1083,7 @@ <h2>
               </dt>
               <dd>
                 <p>
-                  Only [= map/exist =]s for video. It represents the total number of frames correctly decoded
+                  Only defined for video. It represents the total number of frames correctly decoded
```

Comment:
Doesn't not being defined for audio imply that it is undefined?
What was wrong [= map/exists =]?

Reply:
We don't have a definition of framesDecoded for audio which by virtue of "we can only have things which are defined" means this can not exist on audio.

`only defined` is a bit stronger than `only exists`. For example, encoderImplementation only exists on video but only after a frame was encoded.

One can also say the opposite, "Never exists for audio" but... that is [so negative](https://www.youtube.com/watch?v=dQw4w9WgXcQ)

Reply:
I don't want us to use "only defined" language because of the above confusing. I think the metric is "defined" for audio - the definition is: "if audio, this attribute does not exist".

If you really want to avoid the word exist you can say "only applicable to", but I don't see any downsides with being clear about it not existing, since this is in fact different from existing and having the value undefined

Reply:
Much better to say "only exists for video" or "does not exist for audio". Let's say what we mean and not something that is up for interpretation

Reply:
I think this reduces the searchability but with the latest commit one can still search for "Does not exist for"